test/dummy/config/environments/development.rb

```diff
@@ -25,7 +25,7 @@
   config.action_dispatch.best_standards_support = :builtin
 
   # Raise exception on mass assignment protection for Active Record models
-  config.active_record.mass_assignment_sanitizer = :strict
+  # config.active_record.mass_assignment_sanitizer = :strict
```

Comment:
What's up with these being commented out?

Reply:
I've gone ahead and removed the commented out lines.  Without `protected_attributes`, this configuration no longer exists.